Anime-App/Feature/Anime/View/AnimeTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AnimeTableViewCell.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 09/08/2023.
+//
+
+import UIKit
+
+class AnimeTableViewCell: UITableViewCell {
+    static let cellID = "AnimeTableViewCell"
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.spacing = 8
+        stackView.distribution = .fillEqually
+        stackView.alignment = .fill
+        return stackView
+    }()
+    private lazy var title: UILabel = {
+        let title = UILabel()
+        title.translatesAutoresizingMaskIntoConstraints = false
+        return title
+    }()
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    func configure(anime: Anime) {
+        let editedSlugText = anime.attributes?.slug?.replacingOccurrences(of: "-", with: " ")
```

Comment:
This should have a property that hols the entire anime and then this table view cell would have a delegate that communicates and passes this back to the view controller